geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/iterators/BBOXCombiner.scala

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2014 Commonwealth Computer Research, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.locationtech.geomesa.accumulo.iterators
+
+import java.util
+
+import com.vividsolutions.jts.geom.Point
+import com.vividsolutions.jts.io.WKTReader
+import org.apache.accumulo.core.data.{Key, Value}
+import org.apache.accumulo.core.iterators.Combiner
+import org.locationtech.geomesa.utils.geohash.BoundingBox
+
+import scala.collection.JavaConversions._
+
```

Comment:
Nice implementation, though it may be *nice* to have some documentation, at least to describe what this iterator is used for and why. After all this was a nice solution.....